test/images/README.md

```diff
@@ -0,0 +1,25 @@
+This image is the base image for Kubeflow Pipelines [Prow job](https://github.com/kubernetes/test-infra/blob/6555278147dfff550706b41c3f69f41ecf5a8c5a/config/jobs/kubeflow/kubeflow-postsubmits.yaml#L245). Including presubmit and postsubmit jobs.
+
+## How to build the image
+To build this Docker image, run the following Docker command from the directory containing the image's files:
+
+``` 
+docker build -t gcr.io/ml-pipeline-test/test-worker:${WHATEVER_TAG_VERSION} . 
```

Comment:
Sorry for the confusion.

Here I mean you can put whatever tag version you want in the end. For simplicity let's begin with latest